### ISSUE TITLE
A way to completely ignore frames?

### ISSUE THREAD

**Issue by Stooovie** [latenitefilms/MarkerToolbox#32]
Clients and my team almost always comment with minutes and seconds, never with hours and frames. 

"12:20" is 12th minute, 20th second", never 12th second, frame 20.

Marker Toolbox insists on interpreting "12:20" as "second 12, frame 20". I tried adding various things like "Ignore frames. 12:22 means minute 12, second 22, NOT second 12, frame 22" to the prompt to no avail. Any tips?

**Comment by latenitefilms** [comment]
Discussed here: https://mas.to/@stooovie/112286167983563920

**Comment by latenitefilms** [comment]
@Stooovie - Ummm... If you **Process Comments Locally**, if you just have `00:00`, it should treat it as `MM:SS`:

<img width="756" alt="image" src="https://github.com/latenitefilms/MarkerToolbox/assets/22286696/12d455c3-977d-4417-8e32-749fd6e21a40">

**Comment by Stooovie** [comment]
Yes, it works better. You don't get the advantages of AI but for well-ish formatted notes, it does what I want. Not all notes are well-formatted though :)

**Comment by latenitefilms** [comment]
You could just modify the ChatGPT prompt to allow for whatever the client is providing?
